pipeline/stage03_trigger_detection/scripts/minima.py

```diff
@@ -1,29 +1,54 @@
 import neo
 import numpy as np
 import quantities as pq
-from scipy.signal import argrelmin
+from scipy.signal import find_peaks
 import argparse
 from distutils.util import strtobool
 from utils import load_neo, write_neo, remove_annotations
 
 
-def detect_minima(asig, order, interpolation_points, interpolation):
+def detect_minima(asig, order, interpolation_points, interpolation, threshold_fraction,  min_peak_distance):
+        
     signal = asig.as_array()
-    sampling_time = asig.times[1] - asig.times[0]
-
-    t_idx, channel_idx = argrelmin(signal, order=order, axis=0)
-
+    times = asig.times
+    sampling_rate = asig.sampling_rate
+
+    amplitude_span = np.max(signal, axis=0) - np.min(signal, axis=0)
+    threshold = np.min(signal, axis=0) + threshold_fraction*(amplitude_span)
+    threshold_minima = np.max(-signal, axis=0) - threshold_fraction*(amplitude_span)
```

Comment:
I think the most flexible solution to control which minima to include is to have separate `maxima_threshold_fraction` and `minima_threshold_fraction` (basically replacing the `order` parameter). Or do you have another suggestion that we could explore?